app/assets/stylesheets/course/enrol_requests.css.scss

```diff
@@ -0,0 +1,46 @@
+#enrol_request_form {
+  min-width: 190px;
+  width: 190px;
+
+  a {
+    margin: 5px 0px 5px 0px;
+    width: 100%;
+  }
+}
+
+.bold-text {
+  font-weight: bold;
+}
+
+.enrol-request-row {
+
+  .request-approve {
+    margin-right: 5px;
+  }
+
+  .request-approve, .request-delete {
+    padding-bottom: 2px;
+    padding-top: 2px;
+  }
+
+}
+
+.batch-controls {
+
+  input[type=submit] {
+    // Remove all decorations to look like normal text
+    background: none;
+    border: none;
+    display: inline;
+    font: inherit;
+    margin: 0;
+    padding: 0;
+    outline: none;
+    outline-offset: 0;
+    // Additional styles to look like a link
+    color: #0000ff;
```

Comment:
Color literals like `#0000ff` should only be used in variable declarations; they should be referred to via variable everywhere else.